source/index.html.md

```diff
@@ -36,16 +51,36 @@ The [official documentation](https://www.stellar.org/developers/guides/anchor/in
 
 In this tutorial you will use Stellar to build a low-cost financial service similar to Venmo and instead of following the approach number two which is already documented in the Stellar website, you'll be maintaining a Stellar account for each customer and also making it transparant to the final user that they are using Stellar.
 
+
+[MOVE: This should go higher than the anchor definition. I'd make this part of the introduction]
+
 The following are some of the goals in this tutorial:
 
 1. The final user won't know about Stellar.
+[EXPLAIN: What are seed keys?]
```

Comment:
good point, rephrased as `2. The final user won't need to worry about managing their Stellar account.`